.gitlab-ci.yml

```diff
@@ -107,6 +107,22 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
 
+.requires-docker: &requires-docker
+  - DOCKER_RETRIES=4
+  - DOCKER_RETRY_SLEEP_S=1
+  - DOCKER_RUNNING=false
+  - while [[ $DOCKER_RETRIES -gt 0 ]]; do
```

Comment:
Can we keep the style of the existing snippet and do `for retry in 1 2 3 4 5`? Two reasons, 1) consistency with existing code and 2) do not add a bash dependency

Reply:
sure